recipes/wily/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "wily" %}
+{% set version = "1.15.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ead7821e3d47c2b0dfe6fabcc26b7f7c7db1a6f595cd23857f1693b1314d7371
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - wily = wily:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip >=19.1
+    - flit >=2.0
+  run:
+    - python
```

Comment:
You also need `>=3.6` here to restrict python 2

Reply:
@scopatz do I just ignore the linter when those restrictions are in place?
It gives this warning https://github.com/conda-forge/staged-recipes/pull/10416#issuecomment-564700699